### PR TITLE
Implement method GC for store

### PR DESCRIPTION
**Pull request by Superice666**
Implement method GC for store

### CODE REVIEW

File: src/main/java/io/github/kawamuray/wasmtime/Store.java [hunk 1]
```diff
@@ -50,6 +50,10 @@ public T data() {
         return (T) storedData();
     }
 
+    public void GC() {
```
[the remark this answers is not in the document]
Comment (kawamuray) [reply src/main/java/io/github/kawamuray/wasmtime/Store.java:53]: And I think we don't need to bridge a java method to call a native method. Let's just make this method `native`

File: wasmtime-jni/src/errors.rs [hunk 2]
```diff
@@ -79,4 +79,4 @@ impl<'a> Desc<'a, JThrowable<'a>> for Error {
             .new_object(ex_class, "(Ljava/lang/String;)V", &[jmsg.into()])?
             .into())
     }
-}
+}
```
Comment (kawamuray) [inline wasmtime-jni/src/errors.rs:82]: please revert this unrelated change.

File: wasmtime-jni/src/io_github_kawamuray_wasmtime_Store/imp.rs [hunk 3]
```diff
@@ -55,4 +55,9 @@ impl<'a> JniStore<'a> for JniStoreImpl {
             JObject::null().into_inner()
         })
     }
+
+    fn native_g_c(env: &JNIEnv, this: JObject) -> Result<(), Self::Error> {
+        let mut store = interop::get_inner::<Store<StoreData>>(&env, this)?;
+        Ok(store.gc())
```
Comment (kawamuray) [inline wasmtime-jni/src/io_github_kawamuray_wasmtime_Store/imp.rs:61]: This works, because `store.gc()` returns `()`, but to avoid misleading readers that this is returning something a meaningful value, let's return unit explicitly.

```rust
store.gc();
Ok(())
```

**Review by kawamuray — CHANGES_REQUESTED** [review]
Thanks for the PR! left some comments, PTAL

**Comment by Superice666** [comment]
@kawamuray ok, PTAL, thx

**Review by kawamuray — APPROVED** [review]
LGTM, thanks!
